packages/sitecore-jss-dev-tools/src/resolve-scjssconfig.ts

```diff
@@ -14,9 +14,13 @@ export interface ScJssConfig {
   [configName: string]: JssConfiguration;
 }
 
-export function resolveScJssConfig(configName = 'sitecore', assert = true): Promise<ScJssConfig> {
+export function resolveScJssConfig({ 
```

Comment:
Yeah, this would be a breaking change - but I wonder how many folks are using this function directly? We would be able to provide backwards compatibility by leaving the signature and adding the `configPath` parameter at the end, but this makes things overly verbose (having to specify each parameter in our internal calls). Switching to a named/object parameter certainly seems like the better direction long-term.